src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -1126,6 +1133,11 @@ public function move($blnIsAjax=false)
 			// Redirect or reload
 			if (!$objUploader->hasError())
 			{
+				if ($blnIsAjax)
+				{
+					throw new ResponseException(new Response(\Message::generateUnwrapped(), 201));
```

Comment:
Sure we can overcomplicate things and send headers that noone cares about… Or we can just accept that it does not matter at all and the effective request URL is the folder where files are uploaded to and that should be enough…